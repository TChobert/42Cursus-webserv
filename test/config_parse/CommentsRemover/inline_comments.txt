key=value # only a comment
another=more data#another comment
#start with comment
valid=ok
