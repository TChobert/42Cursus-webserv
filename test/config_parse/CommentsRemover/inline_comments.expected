key=value 
another=more data
valid=ok
